hack/update-codegen.sh

```diff
@@ -22,6 +22,22 @@ GO_CMD=${1:-go}
 KUEUE_ROOT=$(realpath $(dirname ${BASH_SOURCE[0]})/..)
 CODEGEN_PKG=$($GO_CMD list -m -f "{{.Dir}}" k8s.io/code-generator)
 
+source "${CODEGEN_PKG}/kube_codegen.sh"
+
+# Generating conversion and defaults functions
+kube::codegen::gen_helpers \
+  --input-pkg-root sigs.k8s.io/kueue/apis/api_extension \
+  --output-base "$(dirname "${BASH_SOURCE[0]}")/../../.." \
```

Comment:
```suggestion
  --output-base  "${KUEUE_ROOT}" \
```
I think we can simplify as below

Reply:
Fixed